memory/manager/manager_test.go

```diff
@@ -20,6 +20,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
+//nolint:unused
```

Comment:
I agree with this workaround, however please raise an Issue

Reply:
Done, see #270.